ChangeLog.d/psa-crypto-hmac-drbg.txt

```diff
@@ -0,0 +1,5 @@
+Features
+   * The PSA crypto subsystem can now use HMAC_DRBG instead of CTR_DRBG.
+     CTR_DRBG is used by default if it is available, but you can override
+     this choice by setting MBEDTLS_PSA_HMAC_DRBG_MD_TYPE at compile time.
+     Fix #3354.
```

Comment:
By the way, I'm deliberately not mentioning external RNG support in the changelog in this PR. I'll add a changelog entry in the follow-up that makes an external PSA RNG usable from TLS code.